website/docs/quickstart_guide/debian.md

```diff
@@ -2,4 +2,37 @@
 sidebar_position: 2
 ---
 
-# Debian
+# Debian package
+
+To install the .deb packages for FerretDB on your Debian, Ubuntu, Linux, and other Unix-like systems, you can use `apt` or `dpkg`.
+
+Download the latest FerretDB .deb package from [our release pages](https://github.com/FerretDB/FerretDB/releases).
+
+To install FerretDB .deb package using `dpkg`, run the following command in your terminal:
+
+```sh
+$ sudo apt update
+# Install using dpkg
+$ sudo dpkg -i <filename>.deb
+```
+
+It’s important to note that `dpkg` doesn’t address dependencies.
+If you encounter any errors related to dependencies when installing, you can resolve them using the following command:
+
+```sh
+sudo apt install -f
+```
+
+Instead of using `dpkg`, you can use apt to manage, install, and resolve all package dependencies automatically.
+However, you must specify the full path to the .deb file.
+This will stop `apt` from downloading and installing from Ubuntu’s repositories.
+
+In the root repository of the debian file, you should indicate the path by adding "`./`".
+
+```sh
+$ sudo apt update
+# Install via apt
+$ sudo apt install ./<filename>.deb
+```
+
+Once FerretDB is installed, you can start the software.
```

Comment:
Should we add a link how to start it or something?

Reply:
We don't currently have any page on running the software, and it wouldn't be appropriate to link to a blog.

Reply:
Ok! It sounds to me like we need such docs... But I guess we have some plans for it, right?

Reply:
I resolve this thread as we discussed it in person